osi_logicallane.proto

```diff
@@ -0,0 +1,516 @@
+        // larger T values than points on the source lane (at the same S
+        // position). In reference line direction (but not necessarily in
+        // driving direction), this means changing from right to left is
+        // allowed.
+        //
+        PASSING_RULE_INCREASING_T = 2;
+
+        // Only passing in decreasing T direction allowed. Passing is allowed
+        // from one lane to the other if the points on the target lane have
+        // smaller T values than points on the source lane (at the same S
+        // position). In reference line direction (but not necessarily in
+        // driving direction), this means changing from left to right is
+        // allowed.
+        //
+        PASSING_RULE_DECREASING_T = 3;
+
+        // Passing is allowed in both directions (left to right and right to
+        // left).
+        //
+        PASSING_RULE_BOTH_ALLOWED = 4;
+
+        // Passing rule fits neither of the above categories.
+        //
+        // Example: this type needs to be used if passing depends on the agent
+        // type, e.g. if cars may change lane, but trucks may not.
+        //
+        PASSING_RULE_OTHER = 5;
+    }
+}
+
+//
+// \brief A logical lane in the road network.
+//
+// A logical lane is part of a road. Compared to a physical lane (OSI type
+// \c Lane), its existence doesn't hinge on the existence of road markings. So
+// e.g. a narrow urban road with two driving directions but no road markings
+// in-between would be presented as two LogicalLanes, but only one Lane. So one
+// Lane can consist of multiple LogicalLanes. E.g. on intersections, each
+// driving path is one LogicalLane, but the whole area is one \c Lane of type
+// \c #TYPE_INTERSECTION.
+//
+// Outside of junctions, logical lanes are constructed such that each point on
+// the road belongs to at least one (typically: exactly one) logical lane. So
+// there are no gaps between logical lanes, and no areas that don't belong to a
+// logical lane.
+//
+// If OSI is generated from OpenDRIVE, then LogicalLanes map directly to
+// OpenDRIVE lanes. However, it is allowed to merge multiple consecutive
+// OpenDRIVE lanes with the same type into one OSI LogicalLane: if an OpenDRIVE
+// lane has a single successor, which has the same lane type, and this
+// successor has only one predecessor (so no lane merging or splitting) then
+// the two lanes may be presented as one continuous LogicalLane. This may be
+// done recursively.
+//
+// The reference line pointed to by reference_line_id defines an ST coordinate
+// system for the lane. This ST coordinate system is used to describe positions
+// on the lane.
+//
+message LogicalLane
+{
+    // The ID of the logical lane.
+    //
+    // \note Note ID is global unique.
+    //
+    // \rules
+    // is_globally_unique
+    // \endrules
+    //
+    optional Identifier id = 1;
+
+    // The type of the logical lane.
+    //
+    optional Lane.Classification.Subtype type = 2;
```

Comment:
Decision in Road Model Meeting: Should be separated between logical lane classification and physical lane classification because those types might diverge.

Reply:
Done.